Adding privilege reqs to Jenkins installer for win (#3633)
jenkins-infra/jenkins.io: content/doc/book/installing/index.adoc

=== Windows

The simplest way to install Jenkins on Windows platform is to use Jenkins Windows installer. That program will install Jenkins as a service. Keep in mind that to run Jenkins service you need to have service logon permission.

To dowload LTS and latest releases of Windows installer refer to the Windows section of
link:https://www.jenkins.io/download/#downloading-jenkins[Downloading Jenkins] page.

> After completing download open Windows installer and follow its instructions.

Reviewer: ```suggestion
After completing download open the Windows installer and follow its instructions.
```

> Later in case you will decide to upgrade installed version of Jenkins you may read the
> link:https://www.jenkins.io/blog/2020/08/12/windows-installers-upgrade/#next-steps[Upgrade steps] . As described over there you will have different options for doing such upgrade.

Reviewer: ```suggestion
See the link:https://www.jenkins.io/blog/2020/08/12/windows-installers-upgrade/#next-steps[upgrade steps] when you upgrade to a new release.
```
